modules/browsiRtdProvider.js

```diff
@@ -75,7 +75,7 @@ function collectData() {
       sk: _moduleParams.siteKey,
       sw: (win.screen && win.screen.width) || -1,
       sh: (win.screen && win.screen.height) || -1,
-      url: encodeURIComponent(`${doc.location.protocol}//${doc.location.host}${doc.location.pathname}`),
+      url: `${doc.location.protocol}//${doc.location.host}${doc.location.pathname}`,
```

Comment:
What is the reasoning behind not executing encode anymore?

Reply:
we do encoding for the object later - 
https://github.com/prebid/Prebid.js/blob/2b428bc60f5a4a3e043f1ef465cef969ab9aba7a/modules/browsiRtdProvider.js#L278
this line cause double encoding